alica_tests/include/test_alica.h

```diff
@@ -74,12 +74,13 @@ class AlicaTestFixture : public AlicaTestFixtureBase
         ac->setCommunicator<alicaDummyProxy::AlicaDummyCommunication>();
         ac->setWorldModel<alicaTests::TestWorldModel>();
         ac->setTimerFactory<alicaRosTimer::AlicaRosTimerFactory>();
-        ae = AlicaTestsEngineGetter::getEngine(ac);
-        const_cast<IAlicaCommunication&>(ae->getCommunicator()).startCommunication();
         spinner->start();
         creators = {std::make_unique<alica::ConditionCreator>(), std::make_unique<alica::UtilityFunctionCreator>(),
                 std::make_unique<alica::ConstraintCreator>(), std::make_unique<alica::BehaviourCreator>(), std::make_unique<alica::PlanCreator>()};
-        EXPECT_TRUE(ae->init(std::move(creators)));
+
+        EXPECT_EQ(0,ac->init(std::move(creators)));
```

Comment:
Initializing the context instead of the engine leads to starting the engine and failing some tests, which rely on an engine that is not running.

Reply:
@bjoernschroeder 
Due to the change in AlicaContext::AlicaContext and AlicaContext::Init the flow is now:
- AlicaContext::AlicaContext no more create the AlicaEngine object
- AlicaContext::init Create and init the AlicaEngine object

so the
```c++
AlicaContext* ac = new alica::AlicaContext ...
ae=AlicaTestsEngineGetter::getEngine(ac);   Here the _engine in nullptr
ae->init();
```
is no more possible

@bjoernschroeder if you agree I can change the two tests and keep only one endpoint

Reply:
Keeping only one endpoint will make the tests redundant I think. This test is odd, since it is looking at various internal engine APIs & making a bunch of assumptions about them & not following the normal initialization procedure. But I think the test can be modified to work on another plan instead of master. @bjoernschroeder can you check if it is possible to do this? Basically after engine is initialized in the normal way, switch to a state with a sub plan & do the asserts on that plan instead of master.

Reply:
@triccyx you can disable these tests until then